source/clear-linux/tutorials/wp-install/wp-install.rst

```diff
@@ -3,204 +3,150 @@
 Create a Clear Linux WordPress\* server
 #######################################
 
-Prerequisites
-=============
+This tutorial guides you through installing WordPress components on your |CL| LAMP server. During :ref:`Creating a Clear Linux based web server <web-server-install>`, you created the initial WordPress MySQL database `WordPress` using phpMyAdmin\* and MariaDB\*. Now that the LAMP server is up and running, you can add the WordPress components needed to host a WordPress website on your host system.
```

Comment:
Revert to original for the first part of the sentence, and then revise like so: 
"At the end of :ref:`Creating a Clear Linux based web server <web-server-install>`, you created a WordPress\*-ready database using phpMyAdmin\* and  MariaDB\*. Now that the LAMP server is running, you can add the components required for hosting a WordPress website.

Reply:
Done